### ISSUE TITLE
Navigation bug, android tv

### ISSUE THREAD

**Issue by oathzed** [Stremio/stremio-bugs#639]
**Describe the bug**
Being able to navigate the interface when the side menu is open as well

**To Reproduce**
1. Go to a tv show, select an episode
2. Now press to the left
3. The left menu will open, now press the back button
4. The left menu will stay open, you will now be on the episode list, pres to the right, the left menu will still be open, and you can navigate the episodes.

**Expected behavior**
If you navigate to the left menu and you press back, it should take you to the previous menu BUT the left menu should have closed.

**Screenshots**
https://imgur.com/a/DfmceYC

**Android tv:**
 - OS: android tv

 - Version android 12

**Additional context**

**Comment by enrikotero** [comment]
This makes the interface very hard to navigate with a remote. 

If you press the "back button" on step 2 the problem is the same. You end with an eternal open left menu and the only solution is to exit the app.

**Comment by oathzed** [comment]
No, if you pressed the back button when you're on step 2 you would just go back.. which is the correct thing to have happen

**Comment by TRtomasz** [comment]
Please try latest version
